CONTRIBUTING.md

```diff
@@ -0,0 +1,586 @@
+
+# Submitting Issues
+
+Most issues for Tilezen are housed in the [Tilezen/vector-datasource](https://github.com/Tilezen/vector-datasource) repo. Before opening an issue, be sure to search the repository to see if someone else has asked your question before. If not, go ahead and [open a new issue](https://github.com/Tilezen/vector-datasource/issues/new).
+
+## Submitting technical bugs
+
+When submitting bug reports, please be sure to give us as much context as possible so that we can reproduce the error you encountered. Be sure to include:
+
+- System conditons (operating system, browser, etc), if you're running from source
+- Steps to reproduce
+- Expected outcome
+- Actual outcome
+- Screenshots, if applicable
+- Code that exposes the bug, if you have it (such as a failing test or a basic script)
+
+## Submitting issues around vector tile quality
+
+It's important to get feedback about the quality of local tile results. Your local knowledge will make it easier for us to understand the problem. When submitting issues be sure to include:
+
+- Where in the world you were looking (a placename &/or zoom, latitude, longitude)
+- Your tile query
+- Your expected result
+- Your actual result
+
+
+# Pull Requests Welcome!
+
+## Project standards overview
+
+Tilezen has several miscellaneous standards:
+
+- we follow [PEP8](https://www.python.org/dev/peps/pep-0008/) coding style for Python and use [Flake8](http://flake8.pycqa.org/en/latest/) to enforce those conventions
+- we *love* tests, [check them out](https://github.com/tilezen/vector-datasource/tree/master/integration-test)
+- we use [CircleCI](https://circleci.com/gh/mapzen/vector-datasource) for continuous integration testing
+- we use [semver](http://semver.org/) for package versioning
+
+All  unit tests in a project will be automatically invoked when you commit to an existing project; make sure they exit successfully!
+
+## Active contributors
+
+We'll gladly invite active contributors to become members of the [Tilezen organization](https://github.com/Tilezen). New members will gain direct write permissions, *and with great power comes great responsibility*.
+
+## Project overview (what to change, where)
+
+Generally speaking there are three aspects of developing vector tiles.
+
+- Configuring project setup, see [wiki page](https://github.com/tilezen/vector-datasource/wiki/Mapzen-Vector-Tile-Service)
+- Updating database properties (can be done ahead of time or at runtime)
+- Changing how features are selected from the database (requires TileServer restart)
+
+<div class='alert-message'>
+Yellow call-outs like this are meant to draw your attention to an important idea or distinction you should keep in mind.
+</div>
+
+Map database in Postgres stores data from OpenStreetMap and other projects like Natural Earth and Who's On First.
+
+When data is loaded, database "triggers" calculate if a feature is included in which layer(s), at what "minimum zoom", and other Mapzen specific "mz" properties.
```

Comment:
Agreed on linking to triggers here. It can be hard to find where the pieces are if unfamiliar with the current layout.

About tile and feature, we might want to introduce those terms and define them earlier on, probably as the first part of the overview or maybe a separate section before it?